client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -20,9 +25,64 @@ import {
 } from 'lib/cart-values/cart-items';
 import { requestPlans } from 'state/plans/actions';
 import { getPlanBySlug } from 'state/plans/selectors';
+import { FormCountrySelect } from 'components/forms/form-country-select';
+import getCountries from 'state/selectors/get-countries';
+import { fetchPaymentCountries } from 'state/countries/actions';
 
 const debug = debugFactory( 'calypso:composite-checkout' );
 
+function CountrySelectMenu( {
```

Comment:
Can we move this to the bottom of the file? While I haven't always followed this advice myself, I think it's a good idea to have the first thing in the file be the primary export of the file, and leave secondary exports and private functions lower down (this doesn't work for unhoisted symbols but it will work for functions like this one).